string.c

```diff
@@ -4346,11 +4346,25 @@ rb_str_byteindex_m(int argc, VALUE *argv, VALUE str)
 }
 
 #ifdef HAVE_MEMRCHR
+#define rb_memrchr memrchr
+#else
+static void*
+rb_memrchr(const char *search_str, int chr, long search_len)
```

Comment:
I guess we can define a function with the same name.

```suggestion
#ifndef HAVE_MEMRCHR
static void*
memrchr(const char *search_str, int chr, long search_len)
```

Reply:
Agreed, cleans it up a little. Pushed this and the other changes.